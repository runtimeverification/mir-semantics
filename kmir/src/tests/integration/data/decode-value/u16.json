{
  "bytes": [
    241,
    255
  ],
  "types": [],
  "typeInfo": {
    "PrimitiveType": {
      "Uint": "U16"
    }
  }
}
