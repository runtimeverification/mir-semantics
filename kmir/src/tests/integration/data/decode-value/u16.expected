Integer ( 65521 , 16 , false )
